test/embind/embind.test.js

```diff
@@ -642,7 +642,7 @@ module({
             assert.throws(TypeError, function() { cm.const_ref_adder(0n, 1); });
         });
 
-        if (cm['ASSERTIONS']) {
+        if (cm['_ASSERTIONS']) {
```

Comment:
Not totally new to this PR, but this seems like a risky pattern to me, in that some future test refactor might forget to set the var, and then it would be undefined. How about adding a guard for that, like this? (just one guard, here, seems enough)
```suggestion
        assert(cm['_ASSERTIONS'] !== undefined);
        if (cm['_ASSERTIONS']) {
```

Reply:
I revered the while file.